code/style.css

```diff
@@ -1,4 +1,214 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  background: #ffffff;
+  color: #000;
+  font-family: Arial, Helvetica, sans-serif;
+}
+body{
+margin: 0px;
+}
+
+.site-name{
+  text-align: center;
+}
+
+span{
+  font-size: 15px;
+}
+
+/* rules for the header */
+header{
+  border: solid;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.logo{
+  height: 70px;
+  width: 80px;
+  justify-self: flex-end;
+}
+
+ul{
+  list-style: none;
+  justify-self: start;
+}
+
+a{
+  color: black;
+  text-decoration: none;
+}
+
+/* hover effect for navbar */
+a:hover {
+color: tomato;
+}
+
+/* meda rule to dissapear the site name in header when shown in mobile */
+@media (max-width:767px){
```

Comment:
I can't really follow where your different media queries start and end. A suggestion is to end the css-file with one section for tablet and then one for desktop. Above that you keep all your "default" styling, which should be for mobile (if you design mobile first)

Reply:
great ide, thanks! will think of this for next project.